app/admin/RTMediaAdmin.php

```diff
@@ -1932,10 +1966,11 @@ function rtm_addon_license_notice() {
 			foreach ( $addons as $addon ) {
 				if ( empty( $addon['args']['status'] ) || 'valid' !== $addon['args']['status'] ) {
 					$message = sprintf(
+						// translators: License page link.
```

Comment:
You need to mention placeholder as well in comment. Please do this if it missing at any other places.

`// translators: %s License page link.`